AMW_business/src/test/java/ch/puzzle/itc/mobiliar/business/domain/commons/CommonFilterServicePersistenceTest.java

```diff
@@ -139,4 +146,62 @@ public void test_addSpecialAndJoiningFilterAndCreateQuery(){
         assertThat(query.getParameters().iterator().next().getName(), is("Deploymentparameter0"));
     }
 
+
+    @Test
+    public void test_environmentCaseInsensitiveSearchUpper(){
```

Comment:
Only two lines of code differ between the two tests. Maybe move the code to a separate function and call it with different parameters. It's easier to read and maintain.